lib/browser/api/app.js

```diff
@@ -39,6 +41,10 @@ Object.assign(app, {
   }
 })
 
+app.isPackaged = () => {
+  return ['app', 'app.asar'].some(p => fs.existsSync(path.join(process.resourcesPath, p)))
```

Comment:
I think this could be done without a sync `fs` call by just comparing against `__dirname`.

Reply:
The above seems like a best case to me